.travis.yml

```
@@ -5,12 +5,22 @@ branches:
   - v0
 sudo: false
 before_install:
-  - go get github.com/mattn/goveralls
-  - go get -t ./...
+- make tools
+- go get -t ./...
 script:
-  - go test . -v
-  - go test ./retrodep/glide -v
-  - go test ./retrodep -v -covermode=count -coverprofile=profile.cov
-  - $GOPATH/bin/goveralls -coverprofile=profile.cov -service=travis-ci
+- make check
+- make test
+- make build
+after_success:
+- make coveralls
 notifications:
   email: false
+deploy:
+  provider: releases
+  api_key:
+    secure: YNdJ6dky6u5D1CLn1pYy4ElteRw1I4vq/pYCokCbbf9Gmq37eq6sOB/M1qUrbuyJQlplKGfXJp4ZI4+2aA8XFu6V7uOlZRPGFPNi8FWLichjikCHnZk/C8zT59KAn77IjyHBW2dmuxoobgYLXOqLeLgub8c8K5azgx9dWb4FUFlppZAk4X347im5cDPOwrA6Yl7MkSMi/lEeXFxhynqTTaqszEUvcAtIxHyHJEtThG43zm15Rwe/TCtTKLQPjh/Qh/GfsNg7cbygfZ/yXpbAPCfdPMxdfSSGKFz1oWdrspEFCBQI0ZfzfRFngEEnXqxQErnkcC8ocvxnW06GBgVqc2A5g4BLwNktII9vtOrvoz3zteEyrZgsCVRFnpHyR7U5Sg7t6F2Z5L/H1+PR9iNgkbyU5+pD9usRq3YuThdypDjE8/5n8ksaol2cJvYcSKaM0Lx0cLOGJSK1WrzMw0TtKCzgIIxSext+pHlRWoLxS3JuV4vEySV/NcdB+wP2vgsflTVKC3CNaHhSqoO4wQUdzgz0jfAqZGx2oGxym4LxKmmOoklu6EVILsJhebVZhKsW0w+GnNTd3yunOCTKYZrAdjDCvISkI2E7CovjlCvJ6Z033jxJ4EXo+k99RtnisORiUyjhCm+tQ5X9mXZwGoUMXdI4XtXP53cQk88QZIsHxis=
+  file: bin/retrodep
```

Not sure about using the binary as the release artifact:
- it will be amd64-only
- it will not be signed
- it might encourage trust, whereas this project is about verification :)

WDYT?

---

Think that we should define which target arch's will we need and add them to makefile. We also need to change this section to release all files by glob. 
```suggestion
  file_glob: true
  file: bin/retrodep.*
```

If this fits you, can you please specify which build targets should we support?

---

Looks great! :+1: 

I'd agree with twaugh on not shipping binaries mainly for the 2 latter reasons regarding trust. We could leave that to distributors such as linux distros and container registries (when not being obtained through `go get`).

---

Ok, I can just drop this commit and we can apply only the first commit of this patch. Your opinion @twaugh @athos-ribeiro ?

---

Sounds good!